keps/sig-api-machinery/20190927-insecure-backend-proxy.md

```diff
@@ -220,3 +221,60 @@ Similar to the `Drawbacks` section the `Alternatives` section is used to highlig
 Use this section if you need things from the project/SIG.
 Examples include a new subproject, repos requested, github details.
 Listing these here allows a SIG to get the process for these resources started right away.
+
+## Production Readiness
+
+  * Feature enablement and rollback
+    - How can this feature be enabled / disabled in a live cluster?
+      Once GA, this feature cannot be disabled at a cluster-wide level. It is the decision of the client whether or not
```

Comment:
nit: not sure if there is a way to make the markdown render the answer on a different line.

Reply:
Two spaces after "cluster?" might do that (https://meta.stackexchange.com/a/298872).